Cargo.lock

```diff
@@ -3301,12 +3301,11 @@ dependencies = [
 
 [[package]]
 name = "nu-ansi-term"
-version = "0.46.0"
+version = "0.50.1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "77a8165726e8236064dbb45459242600304b42a5ea24ee2948e18e023bf7ba84"
+checksum = "d4a28e057d01f97e61255210fcff094d74ed0466038633e95017f5beb68e4399"
 dependencies = [
- "overload",
- "winapi",
+ "windows-sys 0.52.0",
```

Comment:
I think we used to get some winapi stuff from here before?

We really should just update our own code from winapi to windows.

Reply:
Until then, I've just enabled the `wincon` feature of `winapi` to fix the build.